ast/ast_gop.go

```diff
@@ -206,3 +206,26 @@ func (p *ForPhraseStmt) End() token.Pos {
 func (*ForPhraseStmt) stmtNode() {}
 
 // -----------------------------------------------------------------------------
+
+// A SliceStep node represents a slice use step increase.
+type SliceStep struct {
```

Comment:
we can call it RangeExpr. It can be: `L:H:S`, `:H:S`, `:H`. So:
- Colon between L:H always exists.
- H always exists.